### PR TITLE
Add Event Properties to Loops Destination

### PR DESCRIPTION
**Pull request by askkaz**
_Our platform now supports using event properties as content in emails. This PR adds the ability to send event properties via Segment._

## Testing

_Tested using the local server and verified that payloads arrive on our platform as expected._

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

https://github.com/segmentio/action-destinations/assets/7975721/e81009fb-6190-4baf-8c35-8e6dbcbc75bc

### CODE REVIEW

File: packages/destination-actions/src/destinations/loops/sendEvent/index.ts [hunk 1]
```diff
@@ -38,6 +38,13 @@ const action: ActionDefinition<Settings, Payload> = {
       default: {
         '@path': '$.userId'
       }
+    },
+    eventProperties: {
+      label: 'Event Properties',
+      description: 'Properties tied to this event, which can be included in emails triggered by this event.',
```
Comment (tcgilbert) [inline packages/destination-actions/src/destinations/loops/sendEvent/index.ts:44]:
```suggestion
      description: 'Event-specific properties that can be included in emails activated by this event.',
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
one minor tweak to the description. otherwise lgtm!

File: packages/destination-actions/src/destinations/loops/sendEvent/index.ts [hunk 2]
```diff
@@ -38,6 +38,13 @@ const action: ActionDefinition<Settings, Payload> = {
       default: {
         '@path': '$.userId'
       }
+    },
+    eventProperties: {
+      label: 'Event Properties',
+      description: 'Event-specific properties that can be included in emails activated by this event.',
```
Comment (askkaz) [inline packages/destination-actions/src/destinations/loops/sendEvent/index.ts:44]:
```suggestion
      description: 'Event-specific properties that can be included in emails triggered by this event.',
```

**Review by tcgilbert — APPROVED** [review]

**Comment by tcgilbert** [comment]
hey @askkaz - thanks for the update. 

I marked this as `ready for release`. The partner engineer that typically deploys these changes is out until Thursday. That said, I've flagged this with engineering to see if we can get this deployed tomorrow. If not, we will get this deployed as soon as our Partner Engineer is back

**Review by varadarajan-tw — APPROVED** [review]

**Comment by varadarajan-tw** [comment]
Hey @askkaz , the `validate` check seems to be failing in CI. Could you re-run `yarn types` and commit additional changes?

**Comment by joe-ayoub-segment** [comment]
hi @askkaz - any chance you could run the following command, and commit the change please?

./bin/run/ generate:types 

We can't deploy unless this is done. Thank you !

**Comment by joe-ayoub-segment** [comment]
hi @askkaz this PR has been deployed. Please check that it works as expected :)

**Pull request merged**